internal/assets/server.go

```diff
@@ -289,9 +290,18 @@ func ProvideAssetServer(ctx context.Context, webMode model.WebMode, webVersion m
 		if err != nil {
 			return nil, errors.Wrap(err, "ProvideAssetServer")
 		}
+		handler := httputil.NewSingleHostReverseProxy(loc)
+		handler.ErrorHandler = func(writer http.ResponseWriter, request *http.Request, e error) {
+			extra := ""
+			if strings.Contains(e.Error(), "connection refused") {
+				extra = `"connection refused" expected on startup in self-built Tilt. Refreshing in a few seconds.<meta http-equiv="refresh" content="2">`
```

Comment:
Can we tell if we're a release or built off of master in the code? If so could we adjust this message appropriately?

Reply:
maybe, with code changes. would it be cool to just say "web-mode=local" instead of getting into whether it's self-built?

Reply:
Yeah totally, that's what I meant to say.